back/sssscs/src/main/java/com/ib/certificate/Certificate.java

```diff
@@ -0,0 +1,58 @@
+package com.ib.certificate;
+
+import java.time.LocalDateTime;
+
+import com.ib.user.User;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "certs")
```

Comment:
Is there a reason this can't be the default table name?

Reply:
Done: [2c9c601](https://github.com/magley/sssscs/pull/3/commits/2c9c6018a9a8a2e1a2b622960ad064172ed0825d)